[IMP] IoT: edits to all IoT docs (#3673)
odoo/documentation: content/applications/productivity/iot/devices/camera.rst

================
Connect a camera
================

> A camera can be used when using the :abbr:`IoT (Internet of Things)` Box with an Odoo database. It

Reviewer: ```suggestion
A camera can be connected to the :abbr:`IoT (Internet of Things)` Box with an Odoo database. It
```

can be added in a few easy steps. Then, it can be used in a manufacturing process and it can be
linked to a control point. Doing so will allow for the taking of pictures when a chosen quality
control point has been reached.

Connection
==========

To connect the camera to the :abbr:`IoT (Internet of Things)` Box, simply connect the two by cable.
This is likely a :abbr:`USB (Universal Serial Bus)` cable.

> If the camera is a `supported one <https://www.odoo.com/page/iot-hardware>`__, there is no need to

Reviewer: ```suggestion
If the camera is `supported <https://www.odoo.com/page/iot-hardware>`__, there is no need to
```

   Picture`. Access *Quality Checks* by navigating to :menuselection:`Quality app --> Quality
   Control --> Quality Checks --> New`.

.. seealso::
   - :doc:`../../../inventory_and_mrp/manufacturing/quality_control/quality_control_points`
   - :doc:`../../../inventory_and_mrp/manufacturing/quality_control/quality_alerts`

Link a camera to a work center in the manufacturing app
=======================================================

To link the camera to an action, it needs to be configured on a work center. Navigate to
:menuselection:`Manufacturing --> Configuration --> Work Centers`. Go to the :guilabel:`Work Center`
the camera will be used in and add the device in the :guilabel:`IoT Triggers` tab under
:guilabel:`Device` by selecting :guilabel:`Add a Line`. Then, it can be linked to the
:guilabel:`Action` labeled :guilabel:`Take a Picture`. A key can be added to trigger the action.

It should be noted that the trigger that is first in the list will be chosen first. So, the order
matters and these triggers can be dragged into order.

.. note::
   On the :guilabel:`work order` screen, a status graphic indicates whether the database is

>    correctly connected to the camera.

Author: Duplicate

Connect a Camera
================

A camera can be used when using the *IoT Box* with an Odoo database. It can be added in a
few easy steps. Then, it can be used in a manufacturing process and it can be linked to a control
point. Doing so will allow for the taking of pictures when a chosen quality control point has been
reached.

Connection
==========

To connect the camera to the *IoT Box*, simply connect the two by cable. This is likely a USB cable.

If the camera is a `supported one <https://www.odoo.com/page/iot-hardware>`__, there is no need to
set up anything as it will be detected as soon as it is connected.

.. image:: camera/camera_01.png
   :align: center
   :alt: Camera recognized on the IoT box.

Link a Camera to a quality control point within a manufacturing process
=======================================================================

In the :menuselection:`Quality app`, a device can be setup on a *Quality Control Point*. Go to the
:menuselection:`Control Points --> Quality Control` and open the control point which will be linked
with the camera.

Now, edit the control point and choose the device from the dropdown list. The changes will

> automatically save when navigating away from the page with the changes.

Reviewer 2: can be misleading as this behavior was introduced in version >= 15. This PR is targeting 14 so it's technically wrong.
Maybe change it to something like "make sure it is saved".

Author: Great Catch, thank you. I've made this change.

Connect a Camera
================

A camera can be used when using the :abbr:`IoT (Internet of Things)` Box with an Odoo database. It
can be added in a few easy steps. Then, it can be used in a manufacturing process and it can be
linked to a control point. Doing so will allow for the taking of pictures when a chosen quality
control point has been reached.

Connection
==========

To connect the camera to the :abbr:`IoT (Internet of Things)` Box, simply connect the two by cable.
This is likely a USB cable.

If the camera is a `supported one <https://www.odoo.com/page/iot-hardware>`__, there is no need to
set up anything as it will be detected as soon as it is connected.

.. image:: camera/camera_01.png
   :align: center
   :alt: Camera recognized on the IoT box.

Link a Camera to a quality control point within a manufacturing process
=======================================================================

> In the :menuselection:`Quality app`, a device can be setup on a *Quality Control Point*. Go to the

Reviewer 3: ```suggestion
In the :menuselection:`Quality` app, a device can be set up on a *Quality Control Point*. Go to the
```

:menuselection:`Control Points --> Quality Control` and open the control point which will be linked
with the camera.

Now, edit the control point and choose the device from the dropdown list. Makes sure the changes are
saved.

.. image:: camera/camera_03.png
   :align: center
   :alt: Setting up the device on the quality control point.

The camera can be used with the selected *Control Point*. When the *Quality Control Point* is

> reached during the manufacturing process the database will prompt the operator to take a picture.

Reviewer 3: ```suggestion
reached during the manufacturing process, the database will prompt the operator to take a picture.
```
